package.json

```diff
@@ -6,46 +6,33 @@
   "license": "GPL-3.0-or-later",
   "repository": "ElrondNetwork/dapp-core",
   "main": "dist/index.js",
-  "module": "dist/dapp-core.esm.js",
-  "typings": "dist/index.d.ts",
+  "typings": "dist/hooks/index.d.ts",
   "engines": {
     "node": ">=16"
   },
   "scripts": {
-    "start": "yarn copy-dev & dts watch",
-    "build": "yarn copy-prod & dts build",
-    "test": "dts test --passWithNoTests",
-    "lint": "dts lint",
-    "prepare": "dts build",
-    "size": "size-limit",
-    "postinstall": "yarn copy-dev",
-    "analyze": "size-limit --why",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "copy-dev": "shx cp src/optionalPackages/dev/* src/optionalPackages",
-    "copy-prod": "shx cp src/optionalPackages/prod/* src/optionalPackages"
+    "start": "npm run watch",
+    "build:types": "tsc --project tsconfig.json",
+    "build": "rimraf dist & node esbuild.js & yarn build:types",
+    "watch": "npm run build -- --watch",
+    "prepare": "npm run build"
   },
   "devDependencies": {
-    "@babel/core": "7.17.8",
-    "@size-limit/preset-small-lib": "7.0.8",
-    "@svgr/rollup": "6.2.1",
     "@testing-library/jest-dom": "5.16.2",
     "@testing-library/react": "^13.3.0",
-    "@tsconfig/create-react-app": "^1.0.2",
-    "@tsconfig/recommended": "^1.0.1",
     "@types/lodash": "4.14.181",
     "@types/lodash.debounce": "4.0.6",
     "@types/lodash.isequal": "4.5.5",
     "@types/lodash.omit": "4.5.6",
     "@types/lodash.throttle": "4.1.6",
+    "@types/platform": "^1.3.4",
+    "@types/qrcode": "^1.4.2",
     "@types/qs": "^6.9.7",
-    "@types/react": "18.0.10",
-    "@types/react-dom": "18.0.5",
     "@typescript-eslint/eslint-plugin": "5.0.0",
     "@typescript-eslint/parser": "5.14.0",
-    "babel-loader": "8.2.4",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "dts-cli": "^1.5.1",
+    "esbuild": "^0.14.43",
```

Comment:
Maybe is better to use a strict package version. Please take into consideration also the other new packages added.